packages-next/types/src/schema/hooks.ts

```diff
@@ -77,7 +77,14 @@ type ResolveInputHook<TGeneratedListTypes extends BaseGeneratedListTypes> = (
 ) =>
   | Promise<TGeneratedListTypes['inputs']['create'] | TGeneratedListTypes['inputs']['update']>
   | TGeneratedListTypes['inputs']['create']
-  | TGeneratedListTypes['inputs']['update'];
+  | TGeneratedListTypes['inputs']['update']
+  // TODO: I'm pretty sure this is wrong, but without these additional types you can't define a
+  // resolveInput hook for a field that returns a simple value (e.g timestamp)
```

Comment:
yeah, this is def wrong. silly me for trusting the docs instead of looking at the code when writing this originally